src/client/main.ts

```diff
@@ -81,9 +121,12 @@ export interface IntentListener {
 }
 
 /**
- * Defines all events that are fired by the service
+ * Defines all events that are fired by the service.
  */
 export type FDC3Event = FDC3MainEvent | FDC3ChannelEvent;
+/**
+ * Type to define the type of events fired by the service.
```

Comment:
"**Type** to define the **type**...." seems a bit awkward. Maybe "Alias for every valid eventType string." or something similar